adb_client.c

```diff
@@ -158,13 +158,9 @@ static void handle_close_frame(adb_client_t *client, apacket *p) {
 
     svc = adb_client_find_service(client, p->msg.arg1, p->msg.arg0);
     if (svc != NULL) {
-        svc->peer_id = p->msg.arg0;
-        adb_service_close(client, svc, p);
```

Comment:
What I learnt while implementing this ADB server is the client provided by google is not as aligned with the specification as we would think so I had to experiment a lot to figure out the right behavior for the server.

I don't remember the purpose of this close message so I'll merge this PR and do some additional testing. Thanks for contributing!